templates/tomcat/server.xml.erb

```diff
@@ -92,14 +92,7 @@
                truststoreFile="<%= scope['::candlepin::truststore_file'] %>"
                keystorePass="<%= scope['::candlepin::keystore_password'] %>"
                keystoreType="<%= scope['::candlepin::keystore_type'] %>"
-               ciphers="SSL_RSA_WITH_3DES_EDE_CBC_SHA,
-                    TLS_RSA_WITH_AES_256_CBC_SHA,
-                    TLS_ECDH_ECDSA_WITH_AES_128_CBC_SHA,
-                    TLS_ECDH_ECDSA_WITH_AES_256_CBC_SHA,
-                    TLS_ECDH_RSA_WITH_3DES_EDE_CBC_SHA,
-                    TLS_ECDH_RSA_WITH_AES_128_CBC_SHA,
-                    TLS_ECDH_RSA_WITH_AES_256_CBC_SHA,
-                    TLS_ECDHE_RSA_WITH_AES_256_CBC_SHA"
+               ciphers="<%= scope['::candlepin::ciphers'].join(",\n                    ") %>"
```

Comment:
Sorry, it looks like you're right. Somehow it looked like 21 to me but that must be something in the way Github renders it. Rechecked it with the original template file and there it's 20 like you have.